templates/look-and-feel/layouts/check_your_answers.html

```diff
@@ -71,6 +71,9 @@
 
       <input class="button" type="submit" value="{{ pageContent.sendApplication | default(defaultContent.sendApplication) }}">
     </form>
+
+    {% block after_form -%}
+    {%- endblock %}
```

Comment:
Shouldn't this block be outside the if statement?

Reply:
@paulgain You're right. I've moved it now